fmt/format.h

```diff
@@ -55,8 +55,16 @@
     (defined(_MSVC_LANG) && _MSVC_LANG > 201402L && _MSC_VER >= 1910)
 # include <string_view>
 # define FMT_HAS_STRING_VIEW 1
+# define FMT_HAS_EXPERIMENTAL_STRING_VIEW 0
 #else
-# define FMT_HAS_STRING_VIEW 0
```

Comment:
I suggest keeping this and removing 2 identical defines below since they don't depend on whether experimental string_view exists.

Reply:
Can't get your point, can you paste here how you want the final code to look like?

Reply:
```
#else
 -# define FMT_HAS_STRING_VIEW 0
 +# if (FMT_HAS_INCLUDE(<experimental/string_view>) && __cplusplus > 201402L)
 +#  include <experimental/string_view>
 +#  define FMT_HAS_EXPERIMENTAL_STRING_VIEW 1
 +# else
 +#  define FMT_HAS_EXPERIMENTAL_STRING_VIEW 0
 +# endif
```